### ISSUE TITLE
(Fixed in KEDR 0.4) Intercept calls to kasprintf and kvasprintf

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/kedr#1]
```
kasprintf() and kvasprintf() declared in <linux/kernel.h> and defined in 
lib/kasprintf.c allocate memory with kmalloc(). These functions are exported 
and are used by a couple of kernel modules.

It should be useful to intercept the calls to these functions, at least in 
LeakCheck.

```

-----
Original issue reported on code.google.com by `euspec...@gmail.com` on 24 Oct 2011 at 10:09

**Comment by GoogleCodeExporter** [comment]
```
The version from the repository now supports processing these functions in 
LeakCheck. 

Other KEDR-based tools do not handle k*asprintf yet. This is because 
kasprintf() has a variable number of arguments which is currently kind of 
tricky to handle in KEDR. In the future versions of KEDR this limitation may be 
removed.
```

-----
Original comment by `euspec...@gmail.com` on 30 Aug 2012 at 7:45

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `euspec...@gmail.com` on 30 Aug 2012 at 7:47
* Changed title: **(Fixed in Hg) Intercept calls to kasprintf and kvasprintf**

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `euspec...@gmail.com` on 27 Sep 2012 at 2:07
* Changed title: **(Fixed in KEDR 0.4) Intercept calls to kasprintf and kvasprintf**
* Changed state: **Fixed**
